### PR TITLE
[k8s][minor] Log minikube version when running integration tests.

### PR DESCRIPTION
**Pull request by vanzin**

### CODE REVIEW

File: resource-managers/kubernetes/integration-tests/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <download-maven-plugin.version>1.3.0</download-maven-plugin.version>
     <exec-maven-plugin.version>1.4.0</exec-maven-plugin.version>
     <extraScalaTestArgs></extraScalaTestArgs>
-    <kubernetes-client.version>4.1.2</kubernetes-client.version>
+    <kubernetes-client.version>4.1.0</kubernetes-client.version>
```
Comment (srowen) [inline resource-managers/kubernetes/integration-tests/pom.xml]: I presume this just needs to be reverted

**Review by srowen — COMMENTED** [review]
Fine of course otherwise

File: resource-managers/kubernetes/integration-tests/pom.xml (hunk 1 above)
Comment (vanzin) [reply resource-managers/kubernetes/integration-tests/pom.xml]: This was an attempt at seeing what's going on in jenkins: see SPARK-26997.

**Comment by vanzin** [comment]
@shaneknapp

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
ok, failure confirmed on my testing box.  this is definitely weird...  i'm hacking around w/your changes and will see if i can figure out what's up.

**Comment by shaneknapp** [comment]
test this please

**Comment by vanzin** [comment]
(I updated my branch to keep the 4.1.2 client after my revert in the master branch, so we can use this to test things still.)

**Comment by shaneknapp** [comment]
> (I updated my branch to keep the 4.1.2 client after my revert in the master branch, so we can use this to test things still.)

alrighty.  i have some time today to look in to both the client + logging stuff.

**Comment by shaneknapp** [comment]
TL;DR:  revert the client changes and this will be good to merge.  i confirmed that the client upgrade breaks the world due to our k8s version.

full update here:  https://issues.apache.org/jira/browse/SPARK-26997?focusedCommentId=16779599&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16779599

**Comment by shaneknapp** [comment]
LGTM++

**Comment by vanzin** [comment]
Merging to master.

(I tried to find a command to also print the k8s version, but couldn't.)

**Comment by shaneknapp** [comment]
> (I tried to find a command to also print the k8s version, but couldn't.)

you and me both, lol
